Dockerfile

```diff
@@ -56,7 +56,6 @@ COPY --from=builder /etc/ssl/certs /etc/ssl/certs
 COPY --from=builder workspace/bin/kcp-front-proxy workspace/bin/kcp workspace/bin/virtual-workspaces /
 COPY --from=builder workspace/bin/kubectl-* /usr/local/bin/
 COPY --from=builder workspace/bin/kubectl /usr/local/bin/
-RUN ln -s /usr/local/bin/kubectl-workspace /usr/local/bin/kubectl-workspaces && ln -s /usr/local/bin/kubectl-workspace /usr/local/bin/kubectl-ws
```

Comment:
This was removed given that's now handled in the Makefile

https://github.com/kcp-dev/kcp/blob/372c93fd537c9a4f38e1b59ee7f099673db8839d/Makefile#L107-L108